syntax = "proto3";

package room;

message CreateRoomRequest {
  string name = 1;
}

message CreateRoomResponse {
  bool success = 1;
  string room_id = 2;
}

message ListRoomsRequest {}

message ListRoomsResponse {
  repeated string room_ids = 1;
}

service RoomService {
  rpc CreateRoom (CreateRoomRequest) returns (CreateRoomResponse);
  rpc ListRooms (ListRoomsRequest) returns (ListRoomsResponse);
}
